### PR TITLE
Update to buildah 1.26.4

### PR DESCRIPTION
**Pull request by ismail**
Drop unneeded 0001-Update-vendor-of-containers-storage.patch

### CODE REVIEW

File: Formula/buildah.rb [hunk 1]
```diff
@@ -1,28 +1,14 @@
...
   depends_on "gpgme"
 
-  # All these patches have been committed upstream, we're just cherry-picking
-  # or backporting them to v1.26.2.
-  patch do
-    url "https://raw.githubusercontent.com/slp/homebrew-krun/cd74e3bd5196134d0219d918651e6b8bbf64ead8/patches/buildah/0001-Update-vendor-of-containers-storage.patch"
-    sha256 "4616b4c41d5c019caa8f691d34f42ea93a98dab30c85535069a85f99f7553829"
-  end
-  patch do
-    url "https://raw.githubusercontent.com/slp/homebrew-krun/cd74e3bd5196134d0219d918651e6b8bbf64ead8/patches/buildah/0002-run_unix-return-a-valid-DefaultNamespaceOptions.patch"
```
Comment (slp) [inline Formula/buildah.rb:19]: We still need this patch...

File: Formula/buildah.rb [hunk 2]
```diff
@@ -1,28 +1,14 @@
...
-    sha256 "02d2c7743e5d200eecd41e0b79e2374635797c38347fc3dd409df51a868c2638"
-  end
-  patch do
-    url "https://raw.githubusercontent.com/slp/homebrew-krun/cd74e3bd5196134d0219d918651e6b8bbf64ead8/patches/buildah/0003-run_unix-don-t-return-an-error-from-getNetworkInterf.patch"
```
Comment (slp) [inline Formula/buildah.rb]: ... and this one too.

File: Formula/buildah.rb [hunk 3]
```diff
@@ -1,28 +1,14 @@
...
+  url "https://github.com/containers/buildah/archive/refs/tags/v1.26.4.tar.gz"
+  sha256 "3c29de975e8e79858a3af6278ebe5efe6140062d92abeee662b96105927d29e9"
   license "Apache-2.0"
 
   depends_on "go" => :build
   depends_on "go-md2man" => :build
   depends_on "gpgme"
 
-  # All these patches have been committed upstream, we're just cherry-picking
-  # or backporting them to v1.26.2.
```
Comment (slp) [inline Formula/buildah.rb:13]: Please keep the comment, updating the version referenced to v1.26.4.

File: Formula/buildah.rb [hunk 4]
```diff
@@ -1,28 +1,14 @@
...
-  patch do
-    url "https://raw.githubusercontent.com/slp/homebrew-krun/cd74e3bd5196134d0219d918651e6b8bbf64ead8/patches/buildah/0001-Update-vendor-of-containers-storage.patch"
```
Comment (slp) [inline Formula/buildah.rb:15]: Indeed, this one can be dropped after updating to v1.26.4. Please also run `git rm patches/buildah/0001-Update-vendor-of-containers-storage.patch` so we drop the patch itself for the repo.

**Review by slp — CHANGES_REQUESTED** [review]
Thanks for the PR! I've put some comments about the patches.

File: Formula/buildah.rb (hunk 3 above)
Comment (ismail) [reply Formula/buildah.rb:13]: Thanks for the great review, sorry I've only compared with the main branch.

**Review by slp — APPROVED** [review]

**Pull request merged**